tequilapi/endpoints/identities.go

```diff
@@ -87,17 +103,17 @@ func (endpoint *identitiesAPI) List(resp http.ResponseWriter, request *http.Requ
 
 // swagger:operation POST /identities Identity createIdentity
 // ---
-// summary: Create new identity
+// summary: Creates new identity
 // description: Creates identity and stores in keystore encrypted with passphrase
 // parameters:
 //   - in: body
 //     name: body
-//     description: Parameters for creating new identity
+//     description: Parameter in body (passphrase) required for creating new identity
```

Comment:
Do we need to specify "in body"? This is transport, we should not couple description to transport.

Reply:
In swagger template this message looks more informative